tests/torch/test_layer_attributes.py

```diff
@@ -154,7 +153,7 @@ def __str__(self):
             groups=2,
             transpose=False,
             padding_values=(0, 0)),
-        metatype_cls=PTDepthwiseConv2dSubtype
```

Comment:
maybe it's worth checking subtype as well

Reply:
As it described [here](https://github.com/openvinotoolkit/nncf/pull/1428/files#diff-eaaa92a602e4cf9ff5c8ed3e761a1d66f4f2cf4f045e3c9e3c7b07902c769aaaR80) models that are just one module aren't replaced by nncf layers. So if module isn't replaced by `PTNNCFConv2d`, it has OP metatype, so it couldn't have `PTDepthwiseConv2d` metatype because it is now the suptype of `PTModuleConv2dMetatype`  and `PTModuleConv2dMetatype` is the subtype of `PTNNCFConv2d`.

Reply:
I believe that logic of metatypes is covered in `tests/torch/test_nncf_network.py`